incubator/prometheus-nats-exporter/templates/deployment.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "prometheus-nats-exporter.fullname" . }}
+  labels:
+    app: {{ include "prometheus-nats-exporter.name" . }}
```

Comment:
lets use the new labels, see https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#names-and-labels

Reply:
Fixed.